steal.lic

```diff
@@ -16,13 +16,25 @@ class Steal
 
     @stealing_options = get_data('stealing').stealing_options.reject { |target| settings.dont_steal_list.include?(target['id'].to_i) }
     @stealing_bag = settings.stealing_bag
-    hometown_data = get_data('town')[settings.hometown]
+    stealing_hometown = settings.hometown 
+    stealing_hometown = settings.force_bin_town if settings.force_bin_town
+    hometown_data = get_data('town')[stealing_hometown]
     @bin_id = hometown_data['thief_bin'] && hometown_data['thief_bin']['id']
     @stealing_towns = settings.stealing_towns
     @bin_stolen = settings.bin_stolen
     if @bin_stolen && !@bin_id
-      DRC.message("*** Binning is not yet supported in #{settings.hometown}")
-      @bin_stolen = false
+      DRC.message("*** Binning check failed first attempt for #{stealing_hometown}. Trying again:")
+      pause 2
+      hometown_data = get_data('town')[stealing_hometown]
+      @bin_id = hometown_data['thief_bin'] && hometown_data['thief_bin']['id']
+      if !@bin_id
+        @bin_stolen = false
+        DRC.message("*** Binning check failed second attempt.  Binning is not yet supported in #{settings.hometown}")
+        pause 10 
```

Comment:
❓ What are the `pause 10` for on lines 33 and 36?

Reply:
1\ Allows timing for whatever issue prevented the data load from occurring to resolve itself.
2\ in case of actual non-bin support for an area, moving on too quickly means the user doesn't see the bad setting.  

Once DRC.message updates are in the second one can possibly be removed, as we can direct that to a window other than main so the user doesn't miss it.

I probably should actually convert this to the DRCT method written at the same time and used in burgle.lic that encapsulates all of this.

Reply:
Gotcha, if the pauses stay then would you mind adding a comment explaining this. Similar to https://github.com/rpherbig/dr-scripts/pull/5095#discussion_r670432665

thanks!

Reply:
The switch to using the DCT.get_hometown_target_id method should take care of this as it's documented in there.